extensions/amp-carousel/0.1/carousel.js

```diff
@@ -155,6 +155,7 @@ export class AmpCarousel extends BaseCarousel {
   doLayout_(pos) {
     this.withinWindow_(pos, cell => {
       this.scheduleLayout(cell);
+      this.scheduleResume(cell);
```

Comment:
Documentation in `base-element.js` indicate pause and resume can be called anytime after `buildCallback`. Calling it after layout finishes make sense too. Nothing is overriding `resumeCallback` yet, so hard to know for sure. @jridgewell thoughts?